test/evp_test.c

```diff
@@ -2260,6 +2252,8 @@ static int encapsulate(EVP_TEST *t, EVP_PKEY_CTX *ctx, const char *op,
     }
     if (EVP_PKEY_encapsulate(ctx, wrapped, &wrappedlen, secret, &secretlen) <= 0) {
         t->err = "TEST_ENCAPSULATE_ERROR";
+        if (t->expected_err && strcmp(t->err, t->expected_err) == 0)
```

Comment:
We dont seem to do this check anywhere else. There are other places that do ret=1 when they set t->err.

Reply:
This for Google's KAT tests, which also test for additional error conditions in parsing, ... that aren't normally part of the KAT tests.

Reply:
When the public key was not to expanded during import, the error was detected in encapsulate, rather during parsing. This is no longer directly needed, but could be useful some day. I've returned to eagerly expanding the public on import.